Library/Formula/vdirsyncer.rb

```diff
@@ -12,13 +12,13 @@ class Vdirsyncer < Formula
     sha256 "d0ad5bd159acdfeaf28aec0abda9f34d35750c4377624a365180c24798a8059c" => :mavericks
   end
 
-  option "without-keyring", "Build without python-keyring support"
+  option "with-remotestorage", "Build with support for remote-storage"
```

Comment:
So make the switch being 'without-remotestorage'? Sure, can do. However, both 0.8.0 and 0.8.1 are already merged, so should I update them or wait until the next release?